src/Services/SchemaPoster.php

```diff
@@ -196,66 +203,26 @@ public function uploadSchemaFile(string $filename, string $file_contents): ?Resp
           'charset' => 'utf8',
         ])
       );
-    $this->getLogger()->debug('Upload url: ' . (string) $uri);
+    $this->logger->debug('Upload url: ' . $uri);
     $request = new Request('POST', $uri, [
       'Accept' => 'application/json',
       'Content-Type' => $content_type,
     ], $file_contents);
-    $response = $this->getClient()->sendRequest($request);
-    $logFunction = in_array($response->getStatusCode(), [200, 201, 202, 203]) ? 'notice' : 'error';
-    $this->getLogger()->{$logFunction}('File: {filename}, Status code: {status_code} - {reason}', [
+    $response = $this->client->sendRequest($request);
+    $this->logger->debug('File: {filename}, Status code: {status_code} - {reason}', [
       'filename' => $filename,
       'status_code' => $response->getStatusCode(),
       'reason' => $response->getReasonPhrase(),
     ]);
-    return $response;
-  }
 
-  /**
-   * Get Client.
-   *
-   * @return \Psr\Http\Client\ClientInterface
-   *   Psr 18 Client, typically PantheonGuzzle instance.
-   */
-  public function getClient(): PSR18Interface {
-    return $this->client;
-  }
-
-  /**
-   * Set Client.
-   *
-   * @param \Psr\Http\Client\ClientInterface $client
-   *   Psr 18 Client, typically PantheonGuzzle instance.
-   */
-  public function setClient(PSR18Interface $client): void {
-    $this->client = $client;
-  }
-
-  /**
-   * Get Logger.
-   *
-   * @return \Psr\Log\LoggerInterface
-   *   PSR Logger interface.
-   */
-  public function getLogger(): LoggerInterface {
```

Comment:
Removing this in favor of `LoggerChannelTrait` (and setting up in the constructor)